src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -311,6 +315,14 @@ trait SubmissionComponent {
       query.result.map(_.toMap)
     }
 
+    def getSubmissionWorkflowStatusCounts(submissionId: UUID): ReadAction[Map[String, Int]] = {
+      val query = for {
+        workflow <- workflowQuery if workflow.submissionId === submissionId
+      } yield (workflow.status)
+
+      query.result.map(wfs => wfs.groupBy(identity).mapValues(_.size))
```

Comment:
this pulls all the workflows into memory then groups and counts, that should all be done in the database